Sources/SwiftDocC/Infrastructure/DocumentationConverter.swift

```diff
@@ -246,6 +246,9 @@ public struct DocumentationConverter: DocumentationConverterProtocol {
         let renderContext = RenderContext(documentationContext: context, bundle: bundle)
         
         try outputConsumer.consume(renderReferenceStore: renderContext.store)
+
+        // Copy images, sample files, and other static assets.
+        try outputConsumer.consume(assetsInBundle: bundle)
```

Comment:
Do you mean the `do-catch` that I removed? I mostly did that because the problem list it passes around doesn't exist yet now that this call happens earlier. And to match the line above…

Reply:
Makes sense. It looks like this entire method is wrapped in a do/catch to catch these errors and emit them as diagnostics so I think this shouldn't actually affect behavior.